edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Service/Constants.cs

```diff
@@ -28,6 +28,7 @@ public static class ConfigKey
             public const string EdgeHubDevServerPrivateKeyFile = "EdgeHubDevServerPrivateKeyFile";
             public const string EdgeHubDevTrustBundleFile = "EdgeHubDevTrustBundleFile";
             public const string EdgeHubClientCertAuthEnabled = "ClientCertAuthEnabled";
+            public const string SslProtocols = "SslProtocols";
```

Comment:
>SslProtocols [](start = 48, length = 12)

We should use uppercase as this is the naming convention. We do have old ones which are lowercase and can't be changed.

Reply:
Are you suggesting any change here?